docs/deployments/deployments.md

```diff
@@ -7,6 +7,7 @@ Deployments are used to group a set of resources that can be deployed as a singl
 ```yaml
 - kind: deployment
   name: <string>  # deployment name (required)
+  python_root: <string>  # path to the root of your python folder that will be appended to PYTHONPATH
```

Comment:
I added "(optional)" here. Do you think that makes sense, or would something like "(default: directory containing cortex.yaml)" make more sense?